chia/harvester/harvester_api.py

```diff
@@ -193,8 +195,19 @@ async def lookup_challenge(filename: Path, plot_info: PlotInfo) -> List[harveste
 
         # Concurrently executes all lookups on disk, to take advantage of multiple disk parallelism
         total_proofs_found = 0
-        for sublist_awaitable in asyncio.as_completed(awaitables):
-            for response in await sublist_awaitable:
+        for filename_sublist_awaitable in asyncio.as_completed(awaitables):
+            filename, sublist = await filename_sublist_awaitable
+            time_taken = time.time() - start
+            if time_taken > 5:
+                self.harvester.log.warning(
+                    f"Looking up qualities on {filename} took: {time.time() - start}. This should be below 5 seconds "
```

Comment:
This is time since the start of the entire load operation, not the individual file lookup, correct? So u could see multiple of these for each file after passing 5 seconds. So maybe rephrase this and use time_taken?

Reply:
Also NAS guys will see this all the time. A lot run at 13 seconds.

Reply:
I don't think so, because all of the lookups start at the same time. Each challenge is sent to a different thread